### ISSUE TITLE
Request for quicker navigation to send volumes in track parameter list

### ISSUE THREAD

**Issue by KeithRempel** [jcsteh/osara#1176]
It would be great to be able to quickly jump through the track parameters to the next send volume. Maybe pressing control down and up arrow could jump between groups of parameters associated with a send or receive?

I was just doing a bunch of mixing with many tracks being send to 2 different reverbs. Having the ability to jump down to the next send or receive volume would save a lot of arrow bashing. I know I could type send volume into the filter, but having a keystroke to zip up and down through the sends and receives would have been great.

Alternatively a tree view style list could be used, with parameters pertaining to a given send or receive grouped together. Arrowing through the many send options feels cumbersome.

Thanks for all of the great work,
Keith

**Comment by jcsteh** [comment]
I can certainly understand the desire for this. That said, control+up/down feels pretty non-standard. I'm also concerned it wouldn't fit well on Mac, where you have to open a combo box before you can interact with it and where we can't control the interaction at all once opened.

A tree view could work, though it would obviously introduce one more arrow stop for each send itself; i.e. send 2 reverb (expanded), 2 reverb send volume, etc.

**Comment by jcsteh** [comment]
Does anyone know if REAPER tree views work well enough on mac?

**Comment by ScottChesworth** [comment]
Going on the category picker in Prefs, they seem to show up as tables.

**Comment by jcsteh** [comment]
Would that be efficient enough in terms of keyboard navigation if we were to use this in the Parameters dialog? Does it support expanding/collapsing with left/right arrow, etc.? I know you can do that with VO, but that seems like it'd be rather inefficient.

**Comment by pitermach** [comment]
Yes, they do, pressing right/left on a branch will expand or collapse it, first letter navigation is supported and just like on Windows if you press left on an item inside a branch the focus is moved to that branch if you want to quickly collapse it.

**Comment by KeithRempel** [comment]
Would an interface where the send volume appears, and the other things are hidden until expanded work?Something like:"Track 4 send volume, press right arrow to expand."I think the send volumes are more likely to be adjusted and tweaked on the fly than the other things pertaining to sends. Hopefully this is in line with others, and not just a bias to my own work flows.On Dec 18, 2024, at 7:32 AM, pitermach ***@***.***> wrote:﻿
Yes, they do, pressing right/left on a branch will expand or collapse it, first letter navigation is supported and just like on Windows if you press left on an item inside a branch the focus is moved to that branch if you want to quickly collapse it.

**Comment by jcsteh** [comment]
That'd be very non-standard for a tree view. It's also weird conceptually: the other send controls aren't "inside" the volume.
